.github/workflows/release.yml

```
@@ -0,0 +1,63 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
```

@tysongach to test you might want to change this to:

```suggestion
on: push
```